packages/editor/src/components/inserter-sidebar/index.js

```diff
@@ -65,6 +68,9 @@ export default function InserterSidebar() {
 						insertionPoint.insertionIndex
 					}
 					__experimentalFilterValue={ insertionPoint.filterValue }
+					__experimentalOnPatternCategorySelection={
+						isRightSidebarOpen ? closeGeneralSidebar() : undefined
```

Comment:
```suggestion
						isRightSidebarOpen ? closeGeneralSidebar : undefined
```

We should pass the function.